system-setup.py

```diff
@@ -22,9 +22,10 @@ def common_first(self):
         self.install("git jq curl")
         self.run("%s/bin/enable-utf8" % READIES)
 
-        self.install("libssl-dev")
-        self.install("autoconf")
-        self.install("libtool")
+        if self.os != 'macos' and self.dist != 'centos':
```

Comment:
```
        if self.os != 'macos' and self.dist != 'centos':
            self.install("libssl-dev autoconf libtool")
```

This is cheaper, because you invoke apt once.

Reply:
I think this commit is not going to work anyway so next commit I'll do it like you suggested, thanks!